index.html

```diff
@@ -577,100 +577,110 @@ <h1 class="main-font font-weight-600 text-black">Our Valued Customers</h1>
-                                        <span>New York, USA</span>
+                                        <a href="https://www.linkedin.com/in/serge-komaromi-917b14154/" class="linkedin-profile">Serge Komaromi</a>
+                                        <span>Geneva, Switzerland</span>
                                     </div>
                                 </div>
                             </a>
                     </li>
 
                     <!-- Fifth Image -->
-                    <li class="items Medical Agencies">
-                        <img src="corporate-finance/images/portfolio-img5.jpg" alt="img">
-                            <a href="corporate-finance/images/portfolio-img5.jpg" class="text-center" data-fancybox="images">
+                    <li class="items Business">
+                        <img src="corporate-finance/images/Besmir.jpeg" alt="img"  class="team-profile">
+                            <a href="corporate-finance/images/Besmir.jpeg" class="text-center" data-fancybox="images">
                                 <div class="overlay">
-                                    <div>
+                                    <div class="overlay-flex">
                                         <div class="search-icon"><i class="fa fa-search"></i></div>
-                                        <h4 class="">Paul Jackson</h4>
-                                        <span>New York, USA</span>
+                                        <a href="https://www.linkedin.com/in/bez-hoxha-pmp-candidate-12307328/" class="linkedin-profile">Besmir Buranaj</a>
+                                        <span>Houston, Texas, United States</span>
                                     </div>
                                 </div>
                             </a>
                     </li>
 
                     <!-- Sixth Image -->
-                    <li class="items Business surgery Medical">
-                        <img src="corporate-finance/images/portfolio-img6.jpg" alt="img">
-                            <a href="corporate-finance/images/portfolio-img6.jpg" class="text-center" data-fancybox="images">
+                    <li class="items Business">
+                        <img src="corporate-finance/images/Ina.jpeg" alt="img" class="team-profile">
+                            <a href="corporate-finance/images/Ina.jpeg" class="text-center" data-fancybox="images">
                                 <div class="overlay">
-                                    <div>
+                                    <div class="overlay-flex">
                                         <div class="search-icon"><i class="fa fa-search"></i></div>
-                                        <h4 class="">Eva Marie</h4>
-                                        <span>New York, USA</span>
+                                        <a href="https://www.linkedin.com/in/inagjikolli/" class="linkedin-profile">Ina Gjikolli</a>

```

Comment:
Ina Gjika is the team name and her Linkedin is here:
https://www.linkedin.com/in/inagjika/